pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1366,14 +1366,27 @@ public class ClassInDefaultPackage {
         <priority>3</priority>
         <properties>
             <property name="version" value="2.0"/>
+            <property name="acceptableDecimalLength" type="Integer" value="4" min="3" max="6"
```

Comment:
Placing the max at 6 seems arbitrary. I think it's not PMD's problem if users want to place it at 10. The rule would be less useful, but it also checks for misplaced underscores if any, so, whatever 

Ideally it would only be required to be positive, but we need to provide a max anyway (refs #1204), which will be fixed in 7.0.0 (#1432). I think we'd better give it a very high value so that it doesn't get in the way for now